index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="Navigation">
+    <div class = "nav-ban bg-dark">
+      <ul class="nb-list">
+        <li class="nb-item"><a class="a-nb-item" href="#"><img class= "fb" src="img/fb.svg"></a></li>
+        <li class="nb-item"><a class="a-nb-item" href="#"><img class= "twtr" src="img/twtr.svg"></a></li>
+        <li class="nb-item"><a class="a-nb-item" href="#">English</a></li>
+        <li class="nb-item"><a class="a-nb-item" href="#">My Page</a></li>
+        <li class="nb-item"><a class="a-nb-item" href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="nav-int bg-white" >
+        
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>        
+        <span class="bar"></span>        
+      </div>
+      <a href="index.html"><img class="logo" src="img/logo.png"></a>
+    
+      <ul class="ni-list">
+        <li class="ni-item"><a class="a-ni-item a-ni-item1" href="about.html">About</a></li>
+        <li class="ni-item"><a class="a-ni-item" href="#">Program</a></li>
+        <li class="ni-item"><a class="a-ni-item" href="#">Join</a></li>
+        <li class="ni-item"><a class="a-ni-item" href="#">Sponsor</a></li>
+        <li class="ni-item"><a class="a-ni-item" href="#">News</a></li>
+        <li class="ni-item"><a class="a-ni-item a-ni-item6" href="#">CC Campaign</a></li>
+      </ul>    
```

Comment:
- [ ] I think it should be better to wrap up your navigation element into a nav tag for accessibility and semantics 👍